pkg/globalmanager/runtime/worker.go

```diff
@@ -170,6 +180,107 @@ func CreatePodWithTemplate(client kubernetes.Interface, object CommonInterface,
 	return createdPod, nil
 }
 
+//CreateEdgeMeshService creates a kubeedge edgemesh service for an object, and returns an edgemesh service URL.
+//Since edgemesh can realize Cross-Edge-Cloud communication, the service can be created both on the cloud or edge side.
+func CreateEdgeMeshService(kubeClient kubernetes.Interface, object CommonInterface, workerType string, inputPort int32) (string, error) {
```

Comment:
What is the difference between CreateEdgeMeshService and CreateKubernetesService? I can't see it.

Reply:
There are three main differences:

1) ObjectMeta uses **Name** instead of **GenerateName** because it relates to the URL returned.

2) A fixed format URL needs to be returned.

3) Port **Name** is required.

Reply:
OK. You can extend CreateKubernetesService to support these differences.